### ISSUE TITLE
Disable animation

### ISSUE THREAD

**Issue by kevinpapst** [kimai/kimai#4486]
### Describe the problem

Some users report that the UI was lagging, due to the complex CSS animations

### Describe the solution you'd like

Add a per-user setting, that leads to adding a new CSS file, which deactivates the animations.

Solutions seems to disable animations: https://github.com/kimai/kimai/discussions/4415

#### Solution

I added it as example here https://www.kimai.org/documentation/plugin-custom-css.html

<img width="701" alt="Bildschirmfoto 2023-11-09 um 15 46 02" src="https://github.com/kimai/kimai/assets/533162/66bfde72-f168-4350-b948-24e4e0b4f963">

**Comment by aldas** [comment]
That blur effect slowdown is also visible/affecting Brave and Firefox under Linux (Wayland+KDE/plasma window manager)

**Comment by kevinpapst** [comment]
It is at the same time funny and super annoying, that a CSS effect that works smoothly on a 5 year old mobile phone is able to slow down entire working stations. What the heck is so complicated about a blur effect? 

Does that happen for Meet/Zoom/whatever video calls as well, when you activate blur?

**Comment by aldas** [comment]
I do not use Meet/Zoom or anything like that. so I can not comment on that.  But I can comment that this slowdown seems to be related to Window size in my case. On 2560x1440 screen and Brave maximized it feels way slower than when I resize window to 30% size and then open modal in Calendar. 

p.s. "fade in" transition time makes it feel psychologically worse. Is it 150ms? it feels way longer.

**Comment by kevinpapst** [comment]
I did some research and it seems that this is slow since it was introduced into the CSS world. Most machines seem to be able to offload that calculation to the GPU and do not show this lagging.

My working machine is an iMac from 2017 and I never had any issues. My mobile phones (here some older Samsung and iPhone) don't have any issues with the backdrop as well, but the modal is maximized there, so this might not count as well. Some older and newer Macbooks don't have any issues, same as many Windows machines I have seen. 

Until now only a handful of people reported this issue. Honestly, and I am saying that even though I used Ubuntu for multiple years as desktop environment myself: Linux and "Windows remote computers" are just edge cases.

Anyway: I am already working on removing the blur altogether. It is looking good, but not worth any more trouble...
